eos-tech-support/eos-core-dump

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -e
+#
+# This program is used to enable core dumps on the system
+
+CORE_DIR="/var/coredumps"
+KERNEL_CORE_PATTERN='core.%t.%E.%p'
+
+# TODO: Implement switches for permanent changes
+if [ $# != 0 ]; then
+  echo "Usage: $0"
+  exit 1
+fi
+
+if [ ! $EUID == 0 ]; then
+  echo "Must be run with sudo privileges! Exiting!"
+  exit 1
+fi
+
+echo "*** Changes do not persist across reboots!! ***"
+echo "*** Changes do not affect daemons!! ***"
+echo
+
+echo "Creating coredump dir in $CORE_DIR (if needed)..."
+mkdir -p $CORE_DIR
+
+echo "Setting/fixing directory permissions..."
+chown root:root "${CORE_DIR}"
+chmod 1777 "${CORE_DIR}"
+
+echo "Changing kernel core settings..."
+sysctl kernel.core_pattern="${CORE_DIR}/${KERNEL_CORE_PATTERN}" 1> /dev/null
+sysctl kernel.core_pipe_limit=4 > /dev/null
+sysctl kernel.core_uses_pid=1 > /dev/null
+sysctl fs.suid_dumpable=1 > /dev/null
+
+echo
+echo "Changing all process ulimits..."
+processes=$(ps aux | sed 1d | awk '{print $2}')
+for process in ${processes}; do
+  echo -n "."
+  # It's possible that the process died between us checking and setting values
+  set +e
+    prlimit --core=unlimited:unlimited --pid $process 2> /dev/null
+  set -e
```

Comment:
@dbnicholson It usually makes the line unreadable, especially with std* redirection. I'll try it out but I think it might end up looking pretty gnarly.